css/admin.css

```diff
@@ -528,6 +528,11 @@ body #post-head-content p.wp-clearfix:nth-child(1) {
 	color: #999;
 }
 
+.select2-dropdown.select2-dropdown--below,
+.select2-dropdown.select2-dropdown--above {
+	margin-top: 0 !important;
+}
```

Comment:
Is this global change necessary? I don't see the difference. Could you please tell me? :)

Reply:
@andernath There has been a change in the process, now it is no longer necessary. thank you